library-model-loader/src/main/resources/edu/ucr/cs/riple/librarymodel/nullable-methods.tsv

```diff
@@ -1,2 +0,0 @@
-test.target.Foo	returnNullableBad(int)
-test.target.Foo	returnNullableGood(int)
```

Comment:
Why are these being removed as part of this PR again?

Reply:
`LibraryModelsLoader` module here is used for unit tests and these are left over contents from previous runs. They shouldn't appear in the git repository.

Reply:
Ok, does that mean we are missing a `.gitignore` entry somewhere?

Reply:
I thought maybe we can keep the file which is more clearer for the reader of the code where library models loader read the data from. `gitignore` will delete the file I guess. There is another approach that skips the file from indexing but that will happen only in my repository not for all clones.

Reply:
If the file is already landed, I don't believe `.gitignore` will remove it, just prevent changes to it from being added to new commits. That said, the alternative is to make sure that, when tests run, whether or not they succeed, the end state of this file is empty.

Reply:
Sure will do that.